hw/top_englishbreakfast/util/prepare_sw.py

```diff
@@ -29,9 +29,9 @@
 ]
 
 BAZEL_BINARIES = [
-    '//sw/device/lib/testing/test_rom',
+    '//sw/device/lib/testing/test_rom:test_rom',
```

Comment:
To make these changes and pacify "flake8" it looks like you have to solve the following problems with this file (which appear to exist in master, but I guess the linting rule is lint changed files, so unless it's somehow changed, the easiest way to make it go away is to fix the file)

https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=77232&view=logs&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=ddf40991-f099-5c77-8b3e-34e18dcd733d&l=24
  flake8 hw/top_englishbreakfast/util/prepare_sw.py
	hw/top_englishbreakfast/util/prepare_sw.py:38:1: E302 expected 2 blank lines, found 1
	hw/top_englishbreakfast/util/prepare_sw.py:44:26: E701 multiple statements on one line (colon)
	hw/top_englishbreakfast/util/prepare_sw.py:50:1: E302 expected 2 blank lines, found 1
	hw/top_englishbreakfast/util/prepare_sw.py:62:1: E302 expected 2 blank lines, found 1
	hw/top_englishbreakfast/util/prepare_sw.py:67:1: E302 expected 2 blank lines, found 1
	hw/top_englishbreakfast/util/prepare_sw.py:140:15: E703 statement ends with a semicolon
	hw/top_englishbreakfast/util/prepare_sw.py:151:1: E305 expected 2 blank lines after class or function definiti

Reply:
I guess it's just a warning...